training/backends/kubernetes.js

```diff
@@ -161,10 +182,35 @@ module.exports = async function execTask(job, spec) {
                                     memory: (Config.TRAINING_MEMORY_USAGE + 100) + 'Mi'
                                 }
                             },
-                            volumeMounts: {}
+                            volumeMounts: [
+                                {
+                                    name: 'config',
+                                    mountPath: '/etc/almond-cloud',
+                                    readOnly: true
+                                },
+                                {
+                                    name: 'training-config',
+                                    mountPath: '/etc/almond-cloud/training',
+                                    readOnly: true
+                                },
```

Comment:
These are site-specific. You can set these in `Config.TRAINING_KUBERNETES_CONTAINER_SPEC_OVERRIDE`, and the volumes in `Config.TRAINING_KUBERNETES_POD_SPEC_OVERRIDE`.